lit_nlp/examples/models/glue_models.py

```diff
@@ -326,7 +326,7 @@ def max_minibatch_size(self):
     return self.config.inference_batch_size
 
   def get_embedding_table(self):
-    return self.vocab, self.model.bert.embeddings.word_embeddings.numpy()
+    return self.vocab, self.model.bert.embeddings.weight.numpy()
```

Comment:
just note to ryan that we'll have to wait to merge this internally until our transformers is updated.